Doc/library/typing.rst

```diff
@@ -1507,8 +1507,8 @@ Other concrete types
    :func:`open`.
 
    .. deprecated-removed:: 3.8 3.12
-      These types are also in the ``typing.io`` namespace, which was
-      never supported by type checkers and will be removed.
+      The ``typing.io`` and ``typing.re`` namespaces are deprecated and will be
+      removed.
```

Comment:
Please also add where people can still get these types from
```suggestion
      removed.  These types should be directly imported from ``typing`` instead.
```

Reply:
Thanks for this note!

Reply:
I have made the requested changes; please review again